bin/git-fork

```diff
@@ -55,6 +90,12 @@ if [ "$origin" = true ]; then
 else
     # clone forked repo into current dir
     git clone "${remote_prefix}${user}/${project}.git" "$project"
+    until [[ -d "$project" ]]; do
+      echo "Github is being slow today. Waiting 10 secs to attempt clone."
```

Comment:
There are various causes that should be responsible for the failure of `git clone`.
Fortunately, the error message from `git clone` is clear enough.
Better shorten the message and remove our guess.